.circleci/config.yml

```diff
@@ -15,82 +15,64 @@ workflows:
   ci:
     jobs:
       # Start with jobs testing various platforms.
-
-      # Every job that pulls a Docker image from Docker Hub needs to provide
-      # credentials for that pull operation to avoid being subjected to
-      # unauthenticated pull limits shared across all of CircleCI.  Use this
-      # first job to define a yaml anchor that can be used to supply a
-      # CircleCI job context which makes Docker Hub credentials available in
-      # the environment.
-      #
-      # Contexts are managed in the CircleCI web interface:
-      #
-      #  https://app.circleci.com/settings/organization/github/tahoe-lafs/contexts
-      - "debian-9": &DOCKERHUB_CONTEXT
-          context: "dockerhub-auth"
-
+      - "debian-9":
+          {}
```

Comment:
Could it be that something, even an empty `{}`, is expected after the `:`? 

I wonder if CircleCI would be pleased with:

```
workflows:
  ci:
    jobs:
      # Start with jobs testing various platforms.
      - "debian-9"
      - "debian-10":
          requires:
            - "debian-9"
      - "ubuntu-20-04"
```

and so on.

Reply:
I half-remember that this is also allowed, yes, but I left all of my yaml gardening tools in my other pants so I'll leave it as-is for now then come back later and delete it all in favor of some GitLab CI configuration.